src/data/nat/gcd.lean

```diff
@@ -466,6 +467,22 @@ by simp [coprime]
 @[simp] theorem coprime_self (n : ℕ) : coprime n n ↔ n = 1 :=
 by simp [coprime]
 
+section big_operators
+
+open_locale big_operators
+
+lemma coprime_prod_left
```

Comment:
Should we not prove these for `unique_factorization_monoid`s rather than just `ℕ`?

Reply:
They are specializions from these: https://leanprover-community.github.io/mathlib_docs/ring_theory/coprime/lemmas.html#is_coprime.prod_left because it's hard to use them at N. Having a single notion of coprime and a single set of lemmas that works everywhere would be great.